### PR TITLE
Fill in missing description metadata

### PR DESCRIPTION
**Pull request by nschonni**
Copied text from project page :smile:

### CODE REVIEW

File: _posts/2013-10-14-canadian-web-experience-toolkit.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 layout: story
 title: Canadian web experience toolkit
-description: ""
+description: "WET-BOEW is an open source code library for building innovative websites that are accessible, usable, interoperable, mobile-friendly and multilingual. This collaborative open source project is led by the Government of Canada."
```
Comment (benbalter) [inline _posts/2013-10-14-canadian-web-experience-toolkit.md]: I'd spell out "Web Experience Toolkit" for the uninitiated.

Comment (nschonni) [inline _posts/2013-10-14-canadian-web-experience-toolkit.md]: Done

**Comment by nschonni** [comment]
@benbalter does this have merge conflicts now?

**Comment by benbalter** [comment]
Awesome. Great addition. Thank you for this.

**Pull request merged**